docs/src/examples/components/Grid/Variations/GridExampleKeyboardNavigable.shorthand.tsx

```diff
@@ -43,31 +43,21 @@ const renderImages = () => {
 
 const renderImageButtons = () => {
   return _.map(imageNames, imageName => (
-    <Button key={imageName} styles={imageButtonStyles}>
+    <Button key={imageName} text styles={imageButtonStyles}>
```

Comment:
would suggest to remove this Button's prop, however I do understand why it was introduced. Still, we shouldn't fine tune the examples because of the particular theme experiencing styling problems - in contrast, we'd better to make them visible.

Imagine what this change would be for any theme other than Teams - the semantics of the markup is not consistent (why `text`) and, most probably, will cause presentation problems to other theme, as it will style `text` Button differently than Teams theme - maybe this `text` Button will have a look drastically different from the default one.

So, based on this reasoning, suggesting to remove this prop from the docs example

Reply:
I removed it. We still need to understand if the fact that the `Button`'s border goes outside the element on keyboard focus is a styling problem. We need to understand if placing `Button` components next to each other is a valid scenario for layouting as shown in: https://codesandbox.io/s/w2xoqpy23l

If that's true, this is a styling issue that need to be addressed in `Button` style file

fyi @bcalvery